components/log/Kconfig

```diff
@@ -70,4 +70,12 @@ menu "Log output"
             bool "System Time"
     endchoice
 
+    config LOG_TIMESTAMP_SOURCE_SYSTEM_DATETIME
```

Comment:
Relatively minor, but for ease of use suggest adding this as a "choice" so the choices become "Milliseconds Since Boot", "System Time", "System Date and Time"

Reply:
It is now another choice in the kconfig.